tests/test_marc_parser.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+from contextlib import nullcontext as does_not_raise
+from datetime import date
+import pickle
+
+from pymarc import Field
+import pytest
+
+from nightshift.datastore import Resource
+from nightshift.marc_parser import BibReader
+
+
+@pytest.mark.parametrize("arg", ["nyp", "bpl"])
+def test_BibReader_library_arg(arg):
+    with does_not_raise():
+        BibReader(marc_fh="foo.mrc", library=arg)
```

Comment:
This is minor but it might be nice to make this `BibReader` instance a test fixture since it gets created in almost all of these tests.

Reply:
Good idea. Added the `fake_BibReader` fixture.